.github/workflows/cicd-production.yaml

```
@@ -0,0 +1,39 @@
+name: CI/CD Production
+
+on:
+  pull_request
+
+jobs:
+  build-and-publish-image:
+    name: Build and publish Docker image
+    runs-on: ubuntu-latest
+    env:
+      REGISTRY_NAME: itemadacr
```

```suggestion
      REGISTRY_NAME: itemadacr.azurecr.io
```
Would simplify further to have the full repo here?

---

Great suggestion!